rust-app/bin-src/main_nanos.rs

```diff
@@ -3,11 +3,14 @@ use rust_app::implementation::*;
 use rust_app::interface::*;
 mod utils;
 
+use ledger_parser_combinators::interp_parser::set_from_thunk;
+
 use core::str::from_utf8;
-use nanos_sdk::buttons::ButtonEvent;
 use nanos_sdk::io;
 use nanos_ui::ui;
 
+use prompts_ui::RootMenu;
+
```

Comment:
Do we have any way we want to order/group these or is this fine?

Reply:
We really should, probably alphabetically, but they aren't consistent anywhere in this codebase yet.